utils/michigan.py

```diff
@@ -0,0 +1,1052 @@
+            column_lst: list of strings of column names
+
+        Returns:
+            filtered_dataframe: filtered dataframe with only the columns
+            listed above
+        """
+        filtered_dataframe = filtered_dataframe[column_lst].copy()
+
+        return filtered_dataframe
+
+    def reorder_columns(
+        self, dataframe: pd.DataFrame, column_lst: list[str]
+    ) -> pd.DataFrame:
+        """Reorders the columns of the inputted dataframe to column order attached
+
+        Inputs:
+            dataframe: dataframe to be sorted
+            column_lst: list of column names for the sort order
+
+        Returns:
+            sorted_dataframe: dataframe in the sorted order
+        """
+        dataframe = dataframe[column_lst].copy()
+
+        return dataframe
+
+    # NOTE: Helper functions for creating the individuals table are below
+
+    def standardize_and_concatenate_individuals(
+        self, individuals: pd.DataFrame, candidates: pd.DataFrame
+    ):
+        """Standardizes and concatenates the individual donors and candidates
+            into one dataframe
+
+        Inputs:
+            individuals: DataFrame with individuals data
+            candidates: DataFrame with candidates data
+
+        Returns: individuals_df: standardized DataFrame with individual data
+            following the database schema
+
+        """
+        individuals["entity_type"] = "Individual"
+        candidates["entity_type"] = "Candidate"
+
+        # rename the columns so they can be concatenated
+        individuals = individuals.rename(columns={"full_name_uuid": "id"})
+        candidates = candidates.rename(
+            columns={
+                "candidate_full_name_uuid": "id",
+                "can_first_name": "first_name",
+                "can_last_name": "last_name",
+                "candidate_full_name": "full_name",
+            }
+        )
+        individuals_df = pd.concat(
+            [
+                individuals,
+                candidates,
+            ],
+            ignore_index=True,
+            sort=False,
+        )
+        individuals_df["party"] = np.nan
+        individuals_df["state"] = individuals_df["state"].fillna("MI")
+
+        return individuals_df
+
+    def create_filtered_individuals_tables(
+        self, standardized_dataframe_lst: list[pd.DataFrame]
+    ) -> list[pd.DataFrame, pd.DataFrame]:
+        """Filters the list of dataframes to create the individuals dataframe
+
+        Inputs:
+            standardized_dataframe_lst: list containing one of dataframes
+            containing standardized Michigan contribution and expenditure data
+
+        Returns:
+            individuals_df: individuals dataframe as defined in the
+            database schema
+            id_mapping: id mapping for the individuals table
+        """
+        merged_dataframe = standardized_dataframe_lst[0]
+
+        individuals_df = self.filter_dataframe(merged_dataframe, "first_name")
```

Comment:
Why are you breaking out individuals vs candidates? A candidate should be an individual... collapsing this would help save quite a few lines and make this more legible